src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,26 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: {
+    user: {
+      id: number;
+      name: string;
+      username: string;
+      email: string;
+    } | null;
+    id: number;
+    title: string,
+    completed: boolean;
```

Comment:
The 'completed' field is present in the 'todos' type definition, but it is not used in the component. If it's not needed, consider removing it to keep the code cleaner.